README.md

```diff
@@ -21,6 +21,7 @@ The following features are currently supported:
 
 * Auth
   * [AppRole](https://www.vaultproject.io/docs/auth/approle)
+  * [AWS](https://www.vaultproject.io/docs/auth/aws)
```

Comment:
Can you also update the header of `lib.rs` with this line? These two files should stay in sync.

Reply:
Updated